packages/expo-image-picker/ios/ImagePickerUtils.swift

```diff
@@ -16,7 +16,9 @@ func asyncMap<ItemsType: Sequence, ResultType>(
 }
 
 /**
- Concurrently maps the given sequence.
+ Concurrently maps the given sequence. todo: Remove - This function is no longer in use since it caused
```

Comment:
If it's no longer in use then we can remove it now, can't we?

Reply:
We could, just wanted to make the change as small as possible... ?

Reply:
I'll remove it.